src/test/rpc_wallet_tests.cpp

```diff
@@ -242,6 +252,35 @@ BOOST_AUTO_TEST_CASE(rpc_wallet)
     BOOST_CHECK(find_value(obj, "founders") == 0.0);
 }
 
+BOOST_AUTO_TEST_CASE(rpc_wallet_getbalance)
+{
+    SelectParams(CBaseChainParams::TESTNET);
+
+    LOCK(pwalletMain->cs_wallet);
+
+    
+    BOOST_CHECK_THROW(CallRPC("z_getbalance too many args"), runtime_error);
+    BOOST_CHECK_THROW(CallRPC("z_getbalance invalidaddress"), runtime_error);
+    BOOST_CHECK_NO_THROW(CallRPC("z_getbalance mhu9XfNv9o9JfTmEoTA6xKJHBfH2BVF8qG"));
```

Comment:
I assume the plan is to have tests for the results of these operations in beta2? Please file a ticket saying how the tests need to be improved.